### ISSUE TITLE
Disable DNSSEC

### ISSUE THREAD

**Issue by TorstenJahnke** [m13253/dns-over-https#21]
Is it possible do disable DNSSEC by switch or command?
I would like not to use this (buggy) feature and all my DNS running without them.

http://www.dnssec-failed.org
https://ianix.com/pub/dnssec-outages.html

I would even never recommend to use this feature.

**Comment by m13253** [comment]
This program does not do DNSSEC validation. Instead, it only passes the "DO" flag from downstream to upstream and "AD" flag from upstream to downstream literally.

In other words,
- If your downstream resolver requires DNSSEC, doh-client will tell upstream to use DNSSEC
- If your downstream resolver disables DNSSEC, doh-client will tell upstream to disable DNSSEC

Note that for some public DNS servers (e.g. Google Public DNS and Cloudflare DNS), DNSSEC is enforced at their side, you can do nothing to change it.

In conclusion, if you want to disable DNSSEC, just do not install a DNSSEC validator. (If you run systemd, disable the builtin validator. If you do not run systemd, DNSSEC is disabled by default unless you turn it on.)
